### DISCUSSION TITLE
希望引引流，那排没啥用

### DISCUSSION THREAD

**Discussion by hk0852** [share121/inter-knot#4943] [原神]
![屏幕截图 2024-09-16 132324](https://github.com/user-attachments/assets/4a018a8b-4899-41d2-b94e-6d39ac7d0750)

**Comment by hk0852** [comment]
因为某些原因，好像不应该直接贴链接

**Comment by hk0852** [comment]
![image](https://github.com/user-attachments/assets/9b30a572-dcf2-481e-b149-285ef607f3e2)

**Comment by hk0852** [comment]
![image](https://github.com/user-attachments/assets/d7361273-161b-4643-b59b-127170ce6cdf)

**Comment by Minapocalis** [comment]
这什么东西？

**Reply by hk0852** [reply]
> 这什么东西？

纳塔的相关音乐和二创，很好听
